llvm/lib/ObjCopy/ELF/ELFObject.h

```diff
@@ -357,6 +357,8 @@ template <class ELFT> class ELFWriter : public Writer {
 
 class BinaryWriter : public Writer {
 private:
+  const uint8_t GapFill;
+  const std::optional<uint64_t> PadTo;
```

Comment:
The `optional` is a bit unnecessary, right? Couldn't you just default `PadTo` to 0?

Reply:
It looks like this is still `optional`?